source/developer/interactive-dialogs.rst

```diff
@@ -240,6 +242,48 @@ The full list of supported fields are included below:
     "placeholder", "String", "(Optional) A string displayed to include a label besides the checkbox. Maximum 150 characters."
     
 
+Radio Element
+^^^^^^^^^^^^^^^^^^^^^^^
+
+In Mattermost version 5.16 and later, you can use radio elements. It looks like a plain text field with a radio button to be selected. Below is an example of a radio element that asks for a department.
+
+.. image:: ../../source/images/interactive-dialog-radio.png
+
+.. code-block:: json
+
+  {
+    "display_name": "What department do you work in?",
+    "name": "department",
+    "type": "radio",
+    "options": [
+      {
+        "text": "Engineering",
+        "value": "engineering"
+      },
+      {
+        "text": "Sales",
+        "value": "sales"
+      },
+      {
+        "text": "Administration",
+        "value": "administration"
+      }
+    ]
```

Comment:
Non blocking: I would also love to see the example above being extended with a `help_text` to ensure that every supported field is shown.